### ISSUE TITLE
[BUG] `static=false` builds and works in terminals that are wide but fails in terminals that are narrow.

### ISSUE THREAD

**Issue by KwatMDPhD** [comonicon/Comonicon.jl#178]
**Describe the bug**
`static=false` builds and works in terminals that are wide but fails in terminals that are narrow.

**Version of the package:**
v0.12.7

**To Reproduce**
1. Build this main state of GSEA.jl https://github.com/KwatMDPhD/GSEA.jl/commit/5fdf5ce6846728bf07893b9d9b7b9c3802ed1b63
2. Run it successfully with a wide window
3. Run it unsuccessfully with narrow window

**Expected behavior**
`static=false` should work with any terminal size.

**Scree
<img width="1775" alt="Screen Shot 2022-02-12 at 10 30 30" src="https://user-images.githubusercontent.com/7741548/153721753-7220b91b-7c2b-4f0e-9786-2dfe4d9c3622.png">
nshots**
<img width="1556" alt="Screen Shot 2022-02-12 at 10 30 53" src="https://user-images.githubusercontent.com/7741548/153721757-2ad667c3-ebd5-46c0-abdc-d98d243d73a2.png">

**Comment by Roger-luo** [comment]
right, the static logic seems not fit it very well, I think probably the best way is to actually find a solution for small width printing than erroring when it's small.
